deploy/019_deploy_bond_token.ts

```diff
@@ -0,0 +1,22 @@
+import "hardhat-deploy";
```

Comment:
This deployment script has not been tested, so may well contain errors.

Reply:
you can simulate this on local hardhat network by running `yarn hardhat deploy --tags BondToken` without specifying the `--network` flag.

Reply:
Resolved: https://github.com/across-protocol/contracts-v2/pull/226/commits/2ae7164d0996e9c752a9ae5c3bee6f493a16bf86.